test/uk/gov/hmrc/ctcguaranteebalanceeisstub/controllers/Generators.scala

```diff
@@ -21,22 +21,23 @@ import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.AccessCode
 import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.GuaranteeReferenceNumber
 import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.requests.GuaranteeReferenceNumberRequest
 import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.responses.AccessCodeResponse
+import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.responses.Balance
 import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.responses.BalanceResponse
 
 trait Generators {
 
-  def balanceResponseGenerator(): Gen[BalanceResponse] = for {
+  val balanceResponse: BalanceResponse = (for {
```

Comment:
No, I meant these could remain as `Gen[...]` fields, just the generators are constant. The values should still be random.